<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tires</class>
 <widget class="QWidget" name="Tires">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(230, 230, 230);
font: 57 16pt &quot;DINPro-Medium&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;DINPro-Light&quot;;</string>
         </property>
         <property name="text">
          <string>Fullscreen</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 57 20pt &quot;DINPro-Medium&quot;;
color: rgb(0, 68, 255);</string>
         </property>
         <property name="text">
          <string>FL %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true"> 
color: rgb(128, 200, 0);
font: 57 20pt &quot;DINPro-Medium&quot;;</string>
         </property>
         <property name="text">
          <string>FR %</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 57 20pt &quot;DINPro-Medium&quot;;
color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>RL %</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 57 20pt &quot;DINPro-Medium&quot;;
color: rgb(255, 170, 0);</string>
         </property>
         <property name="text">
          <string>RR %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;DINPro-Light&quot;;</string>
         </property>
         <property name="text">
          <string>AUTO</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
